Sources/CircularProgress/Vendored/CustomButton.swift

```diff
@@ -240,10 +240,10 @@ open class CustomButton: NSButton {
 		let borderColor = isOn ? color(for: \.activeBorderColor) : color(for: \.borderColor)
 		let shadowColor = isOn ? (activeShadowColor ?? color(for: \.shadowColor)) : color(for: \.shadowColor)
 
-		layer?.animate(color: backgroundColor.cgColor, keyPath: #keyPath(CALayer.backgroundColor), duration: duration)
-		layer?.animate(color: borderColor.cgColor, keyPath: #keyPath(CALayer.borderColor), duration: duration)
-		layer?.animate(color: shadowColor.cgColor, keyPath: #keyPath(CALayer.shadowColor), duration: duration)
-		titleLayer.animate(color: textColor.cgColor, keyPath: #keyPath(CATextLayer.foregroundColor), duration: duration)
+		layer?.animate(color: backgroundColor.cgColor, keyPath: \.backgroundColor, duration: duration)
+		layer?.animate(color: borderColor.cgColor, keyPath: \.borderColor, duration: duration)
+		layer?.animate(color: shadowColor.cgColor, keyPath: \.shadowColor, duration: duration)
+		titleLayer.animate(color: textColor.cgColor, keyPath: \.foregroundColor, duration: duration)
```

Comment:
Now that it's using a real keypath, I think we could change the API to be more Swifty:

```suggestion
		titleLayer.animate(\.foregroundColor, to: textColor.cgColor, duration: duration)
```